exercises/concept/meltdown-mitigation/.docs/hints.md

```diff
@@ -0,0 +1,33 @@
+# Hints
+
+## General
+
+- The Python Docs on [Control Flow Tools][control flow tools] and the Real Python tutorial on [conditionals][real python conditionals] are great places to start.
+- The Python Docs on [Boolean Operations][boolean operations] can be a great refresher on `bools`, as can the Real Python tutorial on [booleans][python booleans].
+- The Python Docs on [Comparisons][comparisons] and [comparisons examples][python comparisons examples] can be a great refresher for comparisions.
+
+## 1. Check for criticality
+
+- Comparison operators and boolean operations can be _combined_.
```

Comment:
This maybe difficult to understand for non native speakers. Maybe a one line example